dist/gaia-header.js

```diff
@@ -677,6 +668,22 @@ return w[n];},m.exports,m);w[n]=m.exports;};})('gaia-header',this));
 },{"./lib/font-fit":4,"gaia-component":1,"gaia-icons":2}],4:[function(require,module,exports){
 ;(function(define){'use strict';define(function(require,exports,module){
 
+  var privMap = new WeakMap();
```

Comment:
Yep right, I can add a comment.

The goal is to hold values for a specific heading.